### DISCUSSION TITLE
Regression - PAT Rubric

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#61] [PAT - Progress Assessment Tools]
| Lesson | Learning Targets | Strengths | Self-Reflection and Feedback |
|---|---|---|---|
| 1 | Get set up for the curriculum by configuring your tools locally and explore Scikit-learn | Test to make sure you can run notebooks and start building using Scikit-learn |  |
| 2 | Prepare your data for a Regression modeling task, learn about visualization | Successfully visualize your data |  |
| 3 | Understand the difference between linear and polynomial regression | Build two models and understand the problems they solve |  |
| 4 | Understand the basics of logistic regression | Build a logistic regression model and understand when to use this tactic |  |

**Comment by pavittarx** [comment]
There is a rubric over here as well https://github.com/microsoft/ML-For-Beginners/blob/main/2-Regression/1-Tools/assignment.md

The one in the assignment didn't make much sense to me. Can you  tell me how should I use these?

**Reply by jlooper** [reply]
hi, here's the thought process. Keep in mind that rubrics in assignments in general are for teachers to assess their students' work, not something for students to worry about except to keep in line with a teacher's expectations. If you are working through the lessons and assignments on your own, use assignment rubrics to keep yourself aligned to expectations of what it means to complete an assignment well.

The PAT tool is different - it's per lesson grouping and helps you to 'learn out loud' if you so choose. You can create a copy of the PAT for Regression, for example, on this discussion board, and annotate it to remind yourself of the things you learned about Regression through the 4 lessons. Hope that helps!

**Reply by pavittarx** [reply]
Hey Jen, 
Thanks for the explaination. This clears it up pretty well for me. :+1:
